### PR TITLE
Create MAINTAINERS.md

### PR DESCRIPTION
**Pull request by demarillacizere**
## What changes were proposed in this pull request?

The change is adding a MAINTAINERS.md document. The list of maintainers was provided by G-Research OSS.

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Current maintainers of the project
+
+| Maintainer             | GitHub ID                                               |
+| ---------------------- | ------------------------------------------------------- |
+| George Jahad           | [GeorgeJahad](https://github.com/GeorgeJahad)           |
```
Comment (GeorgeJahad) [inline MAINTAINERS.md:5]: Hi @demarillacizere I'm George Jahad, the G-Research software engineer listed above.  I know this PR is closed, but I do not want my name listed here.  Please push another branch that removes my name.

Thank you!

**Comment by GeorgeJahad** [comment]
Hi @demarillacizere I believe there has been some kind of misunderstanding.  It is a mistake to have a maintainers file for ozone, especially one that only contains names of current and former GR engineers.

I know this PR is closed but people still look at closed PR's, (that is how I noticed this one.)

Please push a commit to this branch that removes all GR personnel names from the PR.

Thank you!

**Comment by demarillacizere** [comment]
> Hi @demarillacizere I believe there has been some kind of misunderstanding. It is a mistake to have a maintainers file for ozone, especially one that only contains names of current and former GR engineers.
> 
> I know this PR is closed but people still look at closed PR's, (that is how I noticed this one.)
> 
> Please push a commit to this branch that removes all GR personnel names from the PR.
> 
> Thank you!

The maintainers have been removed.

**Comment by GeorgeJahad** [comment]
Thank you!

**Pull request closed without merging**